### ISSUE TITLE
ttItemRef.lua:635: 'for' limit must be a number

### ISSUE THREAD

**Issue by NayooZ** [frozn/TipTac#93]
Happens upon using Feral Spirit (shaman wolves).

```
1x TipTacItemRef\ttItemRef.lua:635: 'for' limit must be a number
[string "@TipTacItemRef\ttItemRef.lua"]:635: in function <TipTacItemRef\ttItemRef.lua:610>
[string "=[C]"]: in function `SetPetAction'
[string "@FrameXML\PetActionBarFrame.lua"]:407: in function `PetActionButton_OnEnter'
[string "*:OnEnter"]:1: in function `OnEnter'
[string "@Bartender4\PetButton.lua"]:25: in function <Bartender4\PetButton.lua:23>

Locals:
self = GameTooltip {
 BottomLeftCorner = <unnamed> {
 }
 NRCSeparator = <unnamed> {
 }
 ttNumLines = 0
 OnBackdropLoaded = <function> defined @SharedXML\Backdrop.lua:152
 GetBackdropColor = <function> defined @SharedXML\Backdrop.lua:390
 SetHyperlink = <function> defined =[C]:-1
 SetTradeSkillItem = <function> defined =[C]:-1
 _owner = QuestieFrame511 {
 }
 ttBackdropBorderColorApplied = false
 questOrder = <table> {
 }
 ttSetBackdropBorderColorLocked = true
 ClearBackdrop = <function> defined @SharedXML\Backdrop.lua:282
 SetPetAction = <function> defined =[C]:-1
 NRCSeparator2 = <unnamed> {
 }
 SetLootItem = <function> defined =[C]:-1
 ttDefaultAnchored = true
 ttGradient = <unnamed> {
 }
 numMoneyFrames = 1
 default = 1
 SetQuestCurrency = <function> defined =[C]:-1
 SetCurrencyTokenByID = <function> defined =[C]:-1
 SetupTextureCoordinates = <function> defined @SharedXML\Backdrop.lua:214
 BottomOverlay = <unnamed> {
 }
 ttLastUpdate = 0
 ttIcon = <unnamed> {
 }
 GetEdgeSize = <function> defined @SharedXML\Backdrop.lua:188
 ttBackdropColorApplied = false
 SetInventoryItem = <function> defined =[C]:-1
 TopEdge = <unnamed> {
 }
 ttCount = <unnamed> {
 }
 __tamedCounts = false
 SetBagItem = <function> defined =[C]:-1
 SetAction = <function> defined =[C]:-1
 SetBackdropColor = <function> defined @TipTac\ttCore.lua:1638
 LeftEdge = <unnamed> {
 }
 SetAuctionItem = <function> defined =[C]:-1
 ttSetIconTextureAndText = <function> defined @TipTacItemRef\ttItemRef.lua:171
 SetSpellBookItem = <function> defined =[C]:-1
 BottomRightCorner = <unnamed> {
 }
 0 = <userdata>
 SetCurrencyToken = <function> defined =[C]:-1
 SetGuildBankItem = <function> defined =[C]:-1
 hasMoney = 1
 TopLeftCorner = <unnamed> {
 }
 SetUnit = <function> defined =[C]:-1
 ttAnchorType = "mouse"
 ttDisplayingUnit = false
 SetCraftItem = <function> defined =[C]:-1
 RightEdge = <unnamed> {
 }
 ttDisplayingAura = false
 ApplyBackdrop = <function> defined @TipTac\ttCore.lua:1622
 SetBackdrop = <function> defined @TipTac\ttCore.lua:1630
 FadeOut = <function> defined @TipTac\ttCore.lua:1435
 TopOverlay = <unnamed> {
 }
 GetBackdropBorderColor = <function> defined @SharedXML\Backdrop.lua:409
 npcOrder = <table> {
 }
 SetUnitDebuff = <function> defined =[C]:-1
 SetItemByID = <function> defined =[C]:-1
 SetBuybackItem = <function> defined =[C]:-1
 SetQuestItem = <function> defined =[C]:-1
 SetMerchantItem = <function> defined =[C]:-1
 SetBorderBlendMode = <function> defined @SharedXML\Backdrop.lua:266
 TopRightCorner = <unnamed> {
 }
 ttUnit = <table> {
 }
 NineSlice = <unnamed> {
 }
 SetLootRollItem = <function> defined =[C]:-1
 SetInboxItem = <function> defined =[C]:-1
 shoppingTooltips = <table> {
 }
 SetBackdropBorderColor = <function> defined @TipTac\ttCore.lua:1646
 manualOrder = <table> {
 }
 SetQuestLogItem = <function> defined =[C]:-1
 OnBackdropSizeChanged = <function> defined @SharedXML\Backdrop.lua:182
 SetUnitBuff = <function> defined =[C]:-1
 BottomEdge = <unnamed> {
 }
 SetTradePlayerItem = <function> defined =[C]:-1
 GetBackdrop = <function> defined @SharedXML\Backdrop.lua:347
 SetSendMailItem = <function> defined =[C]:-1
 ttSetBackdropLocked = true
 HasBackdropInfo = <function> defined @SharedXML\Backdrop.lua:278
 updateTooltip = 0.190000
 TooltipBackdropOnLoad = <function> defined @SharedXML\SharedTooltipTemplates.lua:172
 miniMapIcon = false
 backdropInfo = <table> {
 }
 SetUnitAura = <function> defined =[C]:-1
 SetTradeTargetItem = <function> defined =[C]:-1
 comparing = false
 ttSetBackdropColorLocked = true
 needsReset = true
 SetupPieceVisuals = <function> defined @SharedXML\Backdrop.lua:246
 ttAnchorPoint = "BOTTOMLEFT"
 SetQuestLogCurrency = <function> defined =[C]:-1
 Center = <unnamed> {
 }
}
slot = 1
name = "PET_ACTION_ATTACK"
texture = "PET_ATTACK_TEXTURE"
isToken = 1
isActive = true
autoCastAllowed = false
autoCastEn
```

**Comment by frozn** [comment]
Thanks for reporting this! 👍

Fixed with new release [v22.09.17](https://github.com/frozn/TipTac/releases/tag/v22.09.17).

**Comment by NayooZ** [comment]
Thankyou!
